oasislmf/schema/analysis_settings.json

```diff
@@ -166,6 +166,18 @@
                                 "description": "Period Average Loss Table (ORD Output flag)",
                                 "default": false
                             },
+			    "alct_convergence": {
+				"type": "boolean",
+				"title": "ALCT",
+				"description": "Average Loss Convergence Table (ORD Output flag), requires alt_period set to true",
+				"default": false
+			    },
+			    "alct_confidence": {
+				"type": "float",
```

Comment:
the type `float` needs switching to `number`, just a quirk of OpenAPI https://swagger.io/docs/specification/data-models/data-types/#numbers

Its also worth adding min and max values to limit the valid range like: 
```
   "minimum": 0.0,
   "maximum": 1.0 
```